back/prisma/schema.prisma

```diff
@@ -170,7 +170,8 @@ model Application {
 
 model Company {
   id                                   String                    @id @default(cuid())
-  siret                                String                    @unique
+  orgId                                String                    @unique
+  siret                                String                    @unique @default(cuid())
```

Comment:
Pourquoi `@default(cuid())` ?

Reply:
D'après la description de la PR : 

> createCompany, quand utilisé seulement avec un vatNumber (étranger seulement autorisé), génère automatiquement un siret par cuid() dans le schema.prisma afin de maintenir la retrocompatibilité avec les fonctions s'appuyant sur Company.siret comme PK. (ex. invite to company, etc)

mais j'avoue que ça me chagrine un peu quand même. Pourquoi ne pas changer la signature des mutations `inviteUserToCompany`, `removeUserFromCompany` et `deleteInvitation` pour utiliser le `orgId` ? Seul `sendMembershipRequest` est publique mais devrait être privé à mon avis.

Reply:
Après quelques tests, supprimer la contrainte not null sur siret et transformer tout ce qui repose encore sur un siret obligatoirement représente trop de diff de code qui se rajouterait sur cette PR. Le refacto sguggéré sera donc fait par la suite.